src/components/AllServicesDropdown/AllServicesTabs.tsx

```diff
@@ -74,25 +80,53 @@ const AllServicesTabs = ({
       onToggle={onToggle}
       toggleText={activeTabTitle}
       role="region"
-      className="pf-v5-u-p-md pf-v5-u-pr-0"
+      className="pf-v5-u-p-md pf-v5-u-pr-0 pf-v5-u-pl-0-on-md"
       ouiaId={'all-services-tabs'}
     >
-      <TabWrapper
-        ouiaId="AllServices-favorites-Tab"
-        onClick={(e) => {
-          handleTabClick?.(e, FAVORITE_TAB_ID);
-        }}
-        eventKey={FAVORITE_TAB_ID}
-        title={
-          <TabTitleText>
-            Favorites
-            <Icon className="chr-c-icon-service-tab pf-v5-u-ml-sm" status="warning" isInline>
-              <StarIcon />
-            </Icon>
-          </TabTitleText>
-        }
-      />
+      <Text className="pf-v5-u-pl-lg pf-v5-u-pr-0 pf-v5-u-pt-0 pf-v5-u-pb-sm" component={TextVariants.p}>
+        Platform
+      </Text>
+      <Divider className="pf-v5-u-pb-md" />
+      <ChromeLink href="/ansible/ansible-dashboard">
+        <Button variant="link" component="a" className="pf-v5-u-pl-lg">
+          Red Hat Ansible Platform
+        </Button>
+      </ChromeLink>
+      <ChromeLink href="/insights/dashboard#SIDs=&tags=">
+        <Button variant="link" component="a" className="pf-v5-u-pl-lg">
```

Comment:
This will throw an invalid DOM nesting error. You can't have `a` tag within `a` tag.

```
Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>
```